loglist2/loglist2_test.go

```diff
@@ -379,6 +379,82 @@ func TestStripInternalSpace(t *testing.T) {
 	}
 }
 
+func TestLogStatesString(t *testing.T) {
+	var tests = []struct {
+		name   string
+		logURL string
+		want   string
+	}{
+		{name: "Frozen", logURL: "https://ct.googleapis.com/aviator/", want: "Frozen"},
+		{name: "Empty", logURL: "https://ct.googleapis.com/icarus/", want: "Empty"},
+		{name: "Retired", logURL: "log.bob.io", want: "Retired"},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			log := sampleLogList.FindLogByURL(test.logURL)
+			got := log.State.String()
+			if got != test.want {
```

Comment:
nit: Could combine the above 2 lines, e.g.
```go
if got := log.State.String(); got != test.want {
```

Reply:
Done.